src/contexts/addresses.tsx

```diff
@@ -32,55 +32,124 @@ import {
 import { NetworkType } from '../utils/settings'
 import { Client, useGlobalContext } from './global'
 
+type TransactionType = 'consolidation' | 'transfer'
+
+type SimpleTx = {
+  txId: string
+  fromAddress: string
+  toAddress: string
+  amount: string
+  timestamp: number
+  type: TransactionType
+  network: NetworkType
+}
+
 export type AddressHash = string
 export type TimeInMs = number
```

Comment:
This should probably be defined out of the `addresses` file, as it can be used globally.